### PR TITLE
Add: Move to yarn

### PR DESCRIPTION
**Pull request by MehulKChaudhari**
This PR will add yarn to the repo.

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -24,5 +24,5 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build
+    - run: yarn ci
```
Comment (prakashchoudhary07) [inline .github/workflows/node.js.yml]: Does yarn supports `yarn ci`?

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (MehulKChaudhari) [reply .github/workflows/node.js.yml]: No just got to know Github does not support and certified yarn.

File: package.json [hunk 2]
```diff
@@ -69,14 +69,14 @@
...
     ],
     "./**/*.{js,jsx,ts,tsx,scss}": [
-      "npm run format"
+      "yarn run format"
     ]
   },
   "volta": {
     "node": "14.19.1",
-    "npm": "6.14.16"
+    "yarn": "1.22.19"
```
Comment (prakashchoudhary07) [inline package.json:80]: We should start using the latest LTS node version. Could you create a new issue for it and track?

**Review by prakashchoudhary07 — DISMISSED** [review]
LGTM

**Comment by prakashchoudhary07** [comment]
Is GitHub workflow updated to yarn, If not create a separate issue and track it.

**Comment by harshith-venkatesh** [comment]
GitHub actions are failing since package lock files are removed.....
Could you please add that changes as well and Prakash's changes requested 
Rest looks good.

**Review by harshith-venkatesh — DISMISSED** [review]

File: package.json (hunk 2 above)
Comment (MehulKChaudhari) [reply package.json:80]: Sure.

File: .github/workflows/node.js.yml [hunk 3]
```diff
@@ -24,5 +24,5 @@ jobs:
       uses: actions/setup-node@v1
       with:
         node-version: ${{ matrix.node-version }}
-    - run: npm ci
-    - run: npm run build
+    - run: yarn install --frozen-lockfile
+    - run: yarn build
```
Comment (prakashchoudhary07) [inline .github/workflows/node.js.yml:28]: Are test commands missing?

Comment (MehulKChaudhari) [reply .github/workflows/node.js.yml:28]: Added.

Comment (MehulKChaudhari) [reply .github/workflows/node.js.yml:28]: We cannot add a test command now. I will make a new PR for it. Some tests are failing we will need to fix those first.

**Review by prakashchoudhary07 — APPROVED** [review]

**Review by shubham-y — APPROVED** [review]

**Pull request merged**